data/json/monsters/insect_spider.json

```
@@ -2021,5 +2021,400 @@
+    "armor_cut": 18,
+    "armor_bullet": 20,
+    "vision_day": 10,
+    "vision_night": 15,
+    "harvest": "arachnid",
+    "reproduction": { "baby_egg": "egg_mole_cricket", "baby_count": 1, "baby_timer": 20 },
+    "baby_flags": [ "SPRING", "SUMMER", "AUTUMN" ],
+    "special_attacks": [ [ "SHRIEK_ALERT", 10 ], [ "SHRIEK_STUN", 5 ], [ "impale", 15 ] ],
+    "anger_triggers": [ "HURT", "SOUND", "STALK" ],
+    "fear_triggers": [ "FIRE" ],
+    "flags": [ "SEES", "BORES", "PUSH_VEH", "HEARS", "KEENNOSE", "PATH_AVOID_DANGER_1", "SMELLS" ]
+  },
+  {
+    "id": "mon_lady_bug_giant",
+    "type": "MONSTER",
+    "name": { "str": "giant lady bug" },
+    "description": "This monster was cute with its black spotted red forewings and was adored by children as well as associated with good luck when it was the normal size, but this is clearly a ferocious predator capable of killing swarms of ants and aphids.",
+    "default_faction": "ladybug",
+    "bodytype": "insect",
+    "species": [ "INSECT" ],
+    "volume": "400 L",
+    "weight": "533 kg",
+    "hp": 350,
+    "speed": 110,
+    "material": [ "iflesh" ],
+    "symbol": "B",
+    "color": "red",
+    "morale": 40,
+    "aggression": 20,
+    "melee_skill": 6,
+    "melee_dice": 2,
+    "melee_dice_sides": 6,
+    "melee_cut": 8,
+    "dodge": 1,
+    "armor_bash": 7,
+    "armor_cut": 20,
+    "armor_bullet": 30,
+    "vision_day": 20,
+    "vision_night": 5,
+    "harvest": "arachnid",
+    "reproduction": { "baby_egg": "egg_lady_bug", "baby_count": 1, "baby_timer": 30 },
+    "baby_flags": [ "SPRING", "SUMMER", "AUTUMN" ],
+    "anger_triggers": [ "PLAYER_CLOSE" ],
+    "fear_triggers": [ "HURT" ],
+    "special_attacks": [
+      {
+        "type": "bite",
+        "cooldown": 4,
+        "accuracy": 6,
+        "damage_max_instance": [ { "damage_type": "stab", "amount": 15, "armor_penetration": 0.6 } ]
+      }
+    ],
+    "flags": [ "SEES", "HEARS", "SMELLS", "FLIES", "PATH_AVOID_FIRE" ]
+  },
+  {
+    "id": "mon_grasshopper_small",
+    "type": "MONSTER",
+    "name": { "str": "huge grasshopper" },
+    "description": "A large grasshopper a few times bigger than a regular one.",
+    "copy-from": "mon_grasshopper_giant",
+    "proportional": { "hp": 0.5, "speed": 1.1 },
+    "volume": "750 ml",
+    "weight": "1 kg",
+    "delete": {
+      "reproduction": { "baby_egg": "egg_grasshopper", "baby_count": 3, "baby_timer": 15 },
+      "baby_flags": [ "SPRING", "SUMMER", "AUTUMN" ]
+    },
+    "upgrades": { "age_grow": 8, "into": "mon_grasshopper_giant" }
+  },
+  {
+    "id": "mon_grasshopper_giant",
+    "type": "MONSTER",
+    "name": { "str": "giant grasshopper" },
+    "description": "A huge green grasshopper about the size of a big dog, with long and strong back legs.  It hops, it hops and never stops.",
+    "default_faction": "insect",
+    "bodytype": "insect",
+    "species": [ "INSECT" ],
+    "volume": "25000 ml",
+    "weight": "33 kg",
+    "hp": 40,
+    "speed": 180,
```

```suggestion
    "speed": 180,
    "attack_cost": 540
```
Same deal as the aphids, some leeway to keep them from being effective at combat if they decide to flip hostile.